client/cards/next.js

```diff
@@ -3,19 +3,14 @@ var h = require('hyperscript')
 module.exports = h('.next',
   h('h1', {style: {'font-size': '400%'}}, 'Next js.la'),
 
-  h('h1', 'Thursday, May 28th at TigerText'),
+  h('h1', 'Thursday, September 28th at TigerText'),
 
   h('div', {style: {'margin-top': '10%'}},
     h('h2', 'Brian Mau'),
     h('h2', 'Peer to Peer Video Streaming with WebRTC')
   ),
 
   h('div', {style: {'margin-top': '10%'}},
```

Comment:
I would have preferred to just comment out the second spot so it's easy to add the next time